openid-caep-interoperability-profile-1_0.md

```diff
@@ -93,6 +98,14 @@ Credential Change
 # Common Requirements {#common-requirements}
 The following requirements are common across all use-cases defined in this document.
 
+## Network layer protection
+* The SSF transmitter MUST offer TLS protected endpoints and shall establish connections to other servers using TLS. TLS connections MUST be set up to use TLS version 1.2 or later.
```

Comment:
"shall" -> "SHALL", or make it MUST to be consistent with the rest of the sentence.